### ISSUE TITLE
Discrepancy of fs::is_file vs. fs::file constructor

### ISSUE THREAD

**Issue by Megamouse** [RPCS3/rpcs3#13328]
On windows, fs::is_file returns false if a file path is tested that exists, but has different cases in the filename part.
But then, contrary to the prior check, creating the fs::file will fail with "Already exists".

Example:
! SYS: 'mypath/ROM10R1.BIN' -> fs::is_file = 1
! SYS: 'mypath/rom10r1.BIN' -> fs::is_file = 0
! SYS: Creating: 'mypath/rom10r1.BIN' -> "Already exists"

**Comment by flydt** [comment]
CaseSensitive is disabled in windows by default
if you want to enable it, need by your self with fsutil command

**Comment by Rex000** [comment]
> CaseSensitive is disabled in windows by default if you want to enable it, need by your self with fsutil command

Can you fix this?

**Comment by flydt** [comment]
> > CaseSensitive is disabled in windows by default if you want to enable it, need by your self with fsutil command
> 
> Can you fix this?

you must do by your self, it's not code issue, it's os setting issue

please ref this: https://learn.microsoft.com/en-us/windows/wsl/case-sensitivity

**Comment by Rex000** [comment]
> 

Quite a weird issue. Never seen a case where something needed to be changed in Windows on the user side for an emulator. I wonder why this never happened with any other emulator.

**Comment by flydt** [comment]
> > 
> 
> Quite a weird issue. Never seen a case where something needed to be changed in Windows on the user side for an emulator. I wonder why this never happened with any other emulator.

It has nothing to do with this, first person talk for code, if you are end user, never need to care about this issue

**Comment by Rex000** [comment]
Actually as you can see in the above reference, according to Megamouse the .pkg for Raystorm HD fails to install and gives this error because of this issue:
![216836313-dd8d9c2c-10f2-42bb-8787-67e030f21349](https://user-images.githubusercontent.com/28770012/218192826-244c21b6-95b9-4f43-a240-a53d1dc0b30d.png)

**Comment by Solreavir** [comment]
Yeah looks like this Error has not been fixed I disable all known AV's on my comp and this Error still came up when I tried to install the dump

**Comment by digant73** [comment]
@Megamouse is that still an issue?

**Comment by digant73** [comment]
so it should be enough fs::is_file is improved to be case insensitive

**Comment by Megamouse** [comment]
PS3 is case sensitive. So that seems like the wrong approach

**Comment by digant73** [comment]
> PS3 is case sensitive. So that seems like the wrong approach

beeing from free bsd (if I'm not wrong), so we cannot do so much on Windows (case insensitive). although is_file = 0 should not happen

**Comment by digant73** [comment]
ticket https://github.com/RPCS3/rpcs3/issues/13327 is a use case for this issue (affecting Windows only)

**Comment by digant73** [comment]
#14195 another use case for this issue (affecting Windows only)
